File: Exemplos_Internet/WindowsForms/# Outros/SharedImageListsComponent/SharedImageListsDemoCS/MyForm.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;

namespace SharedImageListsDemo
{
  public partial class MyForm : Form
  {
    public MyForm()
    {
      InitializeComponent();
    }
    protected override void OnLoad(EventArgs e)
    {
      base.OnLoad(e);
      if (this.ListView1.Items.Count > 0 && this.ListView1.Items[0].SubItems.Count > 0)
      {
        this.ListView1.Items[0].SubItems[1].Text = "Handle: " + this.ListView1.LargeImageList.Handle.ToString("x").ToUpper();
      }
    }
    private void btnShowInheritedForm_Click(object sender, System.EventArgs e)
    {
      MyInheritedForm frm = new MyInheritedForm();
      frm.Show();
    }
  }
}

File: Exemplos_Internet/WindowsForms/# Outros/SharedImageListsComponent/SharedImageListsDemoVB/MyForm.vb
Public Class MyForm

  Protected Overrides Sub OnLoad(ByVal e As System.EventArgs)
    MyBase.OnLoad(e)
    If Me.ListView1.Items.Count > 0 AndAlso Me.ListView1.Items(0).SubItems.Count > 0 Then
      Me.ListView1.Items(0).SubItems(1).Text = "Handle: " & Me.ListView1.LargeImageList.Handle.ToString("x").ToUpper
    End If
  End Sub

  Private Sub btnShowInheritedForm_Click(ByVal sender As Object, ByVal e As System.EventArgs) Handles btnShowInheritedForm.Click
    Dim frm As New MyInheritedForm
    frm.Show()
  End Sub
End Class